GoogleForCreators/web-stories-wp
# Fix Media resizing in edit mode

## Summary

Fixes Media resizing in edit mode.


## Relevant Technical Choices
- Applies the local changes on the wrapper element as well to keep the changes in sync.


## To-do



## User-facing changes
|Before|After|
|------|-----|
| ![before](https://user-images.githubusercontent.com/3294597/116968303-87f4f300-acb4-11eb-9339-02dd0d66e09a.gif) | ![after](https://user-images.githubusercontent.com/3294597/116968320-8f1c0100-acb4-11eb-865f-c7252e720383.gif) |


## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Add a media element and enter edit mode
2. Resize the media smaller in edit mode.
3. Verify there is no jumpiness when resizing and after exiting the edit mode, the media has been resized as expected.

### UAT







This PR can be tested by following these steps:

1.

## Checklist




---



Fixes #6247
